VersionControlUI/src/DiffManager.h

```diff
@@ -51,9 +51,23 @@ namespace VersionControlUI {
 
 struct VersionNodes;
 struct ChangeWithNodes;
-struct DiffSetup;
 class DiffFrame;
 
+struct DiffSetup {
+	Model::TreeManager* newVersionManager_{};
+	Model::TreeManager* oldVersionManager_{};
+	FilePersistence::GitRepository* repository_{};
+
+	QString oldVersion_;
+	QString newVersion_;
+};
+
+struct DiffFramesAndSetup
+{
+	QList<DiffFrame*> diffFrames_;
```

Comment:
Who's responsible for deleting these frames? Where does that happen? Is this something we postponed for later?